### PR TITLE
Create URI-patterns.md

### PR DESCRIPTION
**Pull request by ypesk**

### CODE REVIEW

File: URI-patterns.md [hunk 1]
```diff
@@ -0,0 +1,54 @@
+URI patterns for CIMPLE data
+==============================
+
+This is the documentation of the URI design pattern used by the CIMPLE Knowledge Graph. The following SPARQL query provides the number of instances of each type ([results](https://data.silknow.org/sparql?default-graph-uri=&query=SELECT+DISTINCT+count%28%3Fs%29+as+%3Fnb+%3Ftype%0D%0AWHERE+%7B%0D%0A++%3Fs+a+%3Ftype+.%0D%0A++FILTER+%28contains%28str%28%3Ftype%29%2C+%22erlangen%22%29+%7C%7C+contains%28str%28%3Ftype%29%2C+%22forth%22%29%29%0D%0A%7D%0D%0AGROUP+BY+%3Ftype%0D%0AORDER+BY+DESC%28%3Fnb%29&should-sponge=&format=text%2Fhtml&timeout=0&debug=on&run=+Run+Query+)):
```
Comment (ehrhart) [inline URI-patterns.md]: The query URL points to data.silknow.org and the query itself probably needs to be updated

**Review by ehrhart — CHANGES_REQUESTED** [review]

**Comment by ypesk** [comment]
Edited the query URL to data.cimple.eu

**Comment by rtroncy** [comment]
The PR has been merged (thanks @ehrhart) but I have some comments:

- The values in the Class column should really be the original ontological concept, e.g. not "Claim reviews" but `schema:ClaimReview`
- I prefer the '-' instead of the '_' in the URL construction (which follows the URI slugification rules) which would mean having as group 'claim-review' for example
- I think it is better to use singular other plural in the URL, following again the REST principles as this means the collection of all objects of type X. Hence, this would mean having: /organization; /entity; /claim; /claim-review
- I would not make two different group for /rating and /original-rating ... but use a single group /rating. The property can already distinguish, I presume, whether the value is a normalized rating or an original one, do we have different properties?

**Comment by ypesk** [comment]
Thanks for the comments @rtroncy, I will edit the files. What do you think I should specify in the Class column of entities?

**Comment by rtroncy** [comment]
If the entities are extracted with Spotlight (is this what you meant by DBpedia?) then you can put the top class of the DBpedia ontology, `owl:Thing` (https://dief.tools.dbpedia.org/server/ontology/classes/). If the entities are extracted with another tool (spaCy?) and disambiguated with respect to another KG (e.g. Wikidata?), then similarly, putting the top ontology class.

**Comment by ypesk** [comment]
For now it's using DBpedia Spotlight, I will put `owl:Thing`. Thanks

**Pull request merged**